Team9Project/src/com/indragie/cmput301as1/ExpenseItemAddActivity.java

```diff
@@ -64,6 +130,14 @@ protected void onCreate(Bundle savedInstanceState) {
 
 		currencySpinner = (Spinner)findViewById(R.id.sp_currency);
 		SpinnerUtils.configureSpinner(this, currencySpinner, R.array.currency_array);
+		
+		receiptButton = (ImageButton) findViewById(R.id.btn_receipt);
+		receiptButton.setOnClickListener(new View.OnClickListener() {	
+			@Override
+			public void onClick(View v) {
+				startDialog();				
```

Comment:
Would be nice if `startDialog()` was renamed to something more descriptive. `startImagePickerDialog()` maybe?